src/core/server/saved_objects/service/lib/included_fields.ts

```diff
@@ -9,6 +9,22 @@
 function toArray(value: string | string[]): string[] {
   return typeof value === 'string' ? [value] : value;
 }
+
+export const ROOT_FIELDS = [
+  'namespace',
+  'namespaces',
+  'type',
+  'references',
+  'migrationVersion',
+  'coreMigrationVersion',
+  'updated_at',
+  'originId',
+];
+
+export function getRootFields() {
```

Comment:
Why do we need both a mutable and an imutable way to access the ROOT_FIELDS?

Reply:
🤦 

I'll chalk this up to refactoring. Agree we don't need both, I'll avoid exporting `ROOT_FIELDS` in favor of `getRootFields()`.